### ISSUE TITLE
Feature: Show meeting management links in Submenu

### ISSUE THREAD

**Issue by untitled500** [leits/MeetingBar#587]
### Issue that this suggestion fixes

Especially with the lack of icons and a list format, I find it hard to parse a lot of text to see where to click. I need to scroll to the bottom of the list, hover over quick actions and then click to join or dismiss a meeting.

The submenu exists but it only has a option to Copy meeting link. I either want to Join or Dismiss it.

### Solution description

Option 1: Add join meeting / join current meeting / dismiss meeting into the submenu for each meetings.

Option 2 (preferred): Add separate line items to join / dismiss meetings below the current and the next meeting. We can limit next meetings to events starting within next X minutes that I define. Keyboard shortcuts e.g. J or D to cycle through.

### Alternatives considered

See option

**Comment by leits** [comment]
Thanks for the request!
Added dismissal actions in the event menu to the next release scope. To join an event, you only need to click on the event name in the main menu.

https://github.com/leits/MeetingBar/commit/2531afe5a490c4dfe4a986e0555fe8b8210ef700
